### PR TITLE
Disable test test_upsamplingBiMode2d_xla

### PR DESCRIPTION
**Pull request by wonjoolee95**
As a follow-up to https://github.com/pytorch/xla/pull/4308, I'm working on getting the HLO (required to rebase and rebuild my dev env, so taking a little while). Meanwhile, getting this PR ready to disable the affected test if we decide to disable this.

### CODE REVIEW

File: test/pytorch_test_base.py [hunk 1]
```diff
@@ -278,6 +278,7 @@
         'test_upsamplingBicubic2d_correctness_xla',  # FIXME! Got dtypes torch.float32 and torch.float64
         'test_CTCLoss_no_batch_dim_xla',  # Value out of range
         'test_upsamplingBilinear2d_xla',  # precision on GPU/TPU, slow compilation on CPU
+        'test_upsamplingBiMode2d_xla',  # slow compilation on CPU/GPU
```
Comment (vanbasten23) [inline test/pytorch_test_base.py:281]: can you explain to me why running the test would require the "compilation"?

Comment (wonjoolee95) [reply test/pytorch_test_base.py:281]: This refers to the hlo graph compilation. The newly updated test's hlo graph compilation was too slow, making the CI timeout.

**Comment by wonjoolee95** [comment]
The HLO for `XLA_SAVE_TENSORS_FMT='hlo' XLA_SAVE_TENSORS_FILE='/tmp/save1.hlo' python test/test_nn.py TestNNDeviceTypeXLA.test_upsamplingBiMode2d_antialias_False_align_corners_False_mode_bicubic_xla` is at: https://gist.github.com/wonjoolee95/91459aa755a3a7f254b6fa5bf2f9187a.

**Comment by wonjoolee95** [comment]
The build failure seems to be due to the upstream PR requiring a rebase. Locally when rebasing both the upstream and this PR with latest, the build is succeeding.

**Comment by JackCaoG** [comment]
please rebase :D

**Comment by wonjoolee95** [comment]
Oh this shouldn't be needed anymore, since the upstream can set the `onlyNativeDeviceTypes` tag on the tests. I'll close this.

**Pull request closed without merging**